### PR TITLE
Add support for "performed_by" opslog field

### PR DESCRIPTION
**Pull request by dave-shawley**
This optional field can be set to override the display name of the user that created the log entry. I chose to not set it during creation since the creator of the entry is the authenticated user. I did expose the ability to edit it after the fact.

### CODE REVIEW

File: src/js/views/OperationsLog/Edit.jsx [hunk 1]
```diff
@@ -155,6 +158,17 @@ function Edit({ onCancel, onError, onSuccess, operationsLog }) {
...
         errorMessage={errors?.environment}
       />
+      <Form.Field
+        title={t('operationsLog.recordedBy')}
+        name="performed_by"
+        type="text"
+        required={true}
+        description={t('operationsLog.recordedByDescription')}
+        onChange={onValueChange}
+        value={fieldValues.performed_by || fieldValues.recorded_by}
```
Comment (in-op) [inline src/js/views/OperationsLog/Edit.jsx:168]: `fieldValues.recorded_by` doesn't exist

File: public/locales/en.js [hunk 2]
```diff
@@ -324,6 +324,7 @@ export default {
         recordedAt: 'Recorded At',
         completedAt: 'Completed At',
         recordedBy: 'Recorded By',
+        recordedByDescription: 'User name that recorded this operations log',
```
Comment (in-op) [inline public/locales/en.js:327]: Shouldn't this be / use the language of "performed by"?

**Comment by in-op** [comment]
Ideally, I'd like to see the "performed by" across the frontend:
- a field on the ops log create, set automatically to the current user
- rename the column heading on the table

File: public/locales/en.js (hunk 2 above)
Comment (dave-shawley) [reply public/locales/en.js:327]: good catch ... there are still some lingering "recorded by" references in there. Lemme add a commit to clean that up.

Comment (dave-shawley) [reply public/locales/en.js]: fixed in [18dbd0d](https://github.com/AWeber-Imbi/imbi-ui/pull/112/commits/18dbd0df78f99b1396f3e26ac08860f4c539f4e5)

**Comment by dave-shawley** [comment]
> * a field on the ops log create, set automatically to the current user

Addressed in [5c4daeb](https://github.com/AWeber-Imbi/imbi-ui/pull/112/commits/5c4daebe690e3b7aa548704d7336d5bc86e72c13)

**Pull request merged**